Dockerfile

```diff
@@ -42,15 +42,17 @@ RUN set -e \
     case "$apkArch" in \
         armhf) export GOARM='6' ;; \
         x86) export GO386='387' ;; \
-    esac \
-    && \
-    wget -qO- https://dl.google.com/go/go1.12.9.linux-amd64.tar.gz | tar xvz -C /usr/local && \
+    esac
+
+RUN wget -qO- https://dl.google.com/go/go1.12.9.linux-amd64.tar.gz | tar xvz -C /usr/local && \
```

Comment:
hello, why do we have separate run instructions ?

Reply:
it may not be as small as expected from layers

Reply:
I separated them because with it as one instruction, the Docker build was failing and I didn't have a lot of time to debug it. I think for the new version the Dockerfile will be smaller if we base it on the golang image anyway, so this is probably just going to end up as legacy code.

Reply:
yes I know that for debug it's good, but for prod we either need to merge run instructions or use multi-stage builds

Reply:
Left only one place with separated RUN instructions. It's really difficult to understand what's wrong in such long command.